common/src/nonJvm/kotlin/ConcurrentHashMap.kt

```diff
@@ -0,0 +1,7 @@
+package dev.kord.common
+
+import co.touchlab.stately.collections.ConcurrentMutableMap
+import dev.kord.common.annotation.KordInternal
+
+@KordInternal
+public actual typealias ConcurrentHashMap<K, V> = ConcurrentMutableMap<K, V>
```

Comment:
i just remembered that js is single-threaded, wouldn't `MutableMap` be also ok as the actual type?

Reply:
Doesn't nodejs support multithreading?